### PR TITLE
docs(ADOPTERS.md): added Trifork.

### PR DESCRIPTION
**Pull request by cthtrifork**
Signed-off-by: cthtrifork <cth@trifork.com>

## Hey, I just made a Pull Request!

Update the list of "Adopters" in the Backstage.

Completed [Adopter form](https://info.backstage.spotify.com/public-adopter-hand-raiser-form).

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -217,3 +217,4 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
+| [Trifork](https://trifork.com)                                    | [Squad DevTools](mailto:squad_devtools@loft.com.br)                                                                                                                                                                                                                         | We're using Backstage to give visibility and promote ownership of all our applications, resources and tools. Now moving to use it as a Developer Portal to create applications, AWS resources etc.                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (Rugvip) [inline ADOPTERS.md:220]: Looks like a bit of copy paste slipped through here 😅

**Review by Rugvip — CHANGES_REQUESTED** [review]
Thank you, and welcome! 🎉

File: ADOPTERS.md (hunk 1 above)
Comment (cthtrifork) [reply ADOPTERS.md]: Thanks! VSCode has not saved the file before commiting

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**